### PR TITLE
Website rebranding

### LINKED ISSUE #3: Rebrand prismlauncher.github.io
Enable Actions once rebranding is complete.

### PR DESCRIPTION
**Pull request by jackmawer**
I thought I'd try and help the effort by doing some work towards rebranding the website.
Known issues:
- Release links do not work yet given that there are no releases in the PrismLauncher repo yet
- Logo missing due to not being decided on yet
- Favicon still uses PolyMC logo
- CoC email may not exist yet
- GitHub organisation is pending a name change, which will need to be updated when that is completed
- Matrix channels do not exist yet
- Flatpak release is not yet available
- Many screenshots still use PolyMC branding
- News items still reference PolyMC - should they be deleted or left as-is?

Closes #3

### CODE REVIEW

**Review by initmd — APPROVED** [review]

**Comment by vijfhoek** [comment]
This PR spells the project name as "PrismLauncher", but if I understand correctly it's meant to be spelled "Prism Launcher", with a space.

Additionally, I'd just remove all download links for now and add them one by one as they become available - a lot of packages don't exist yet. Better to have no links than a bunch of dead links on the website :)

File: src/wiki/development/translating.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+  key: Translating PrismLauncher
   parent: Development
   order: 3
 --- 
 
-# Translating PolyMC
+# Translating PrismLauncher
 
-The translation effort for PolyMC is hosted on [Weblate](https://hosted.weblate.org/projects/polymc/polymc/), and information about translating PolyMC is available at <https://github.com/PolyMC/Translations>
+The translation effort for PrismLauncher is hosted on [Weblate](https://hosted.weblate.org/projects/prismlauncher/prismlauncher/), and information about translating PrismLauncher is available at <https://github.com/placeholdermc/Translations>
```
Comment (Lutitious) [inline src/wiki/development/translating.md:10]:
```suggestion
The translation effort for PrismLauncher is hosted on [Weblate](https://hosted.weblate.org/projects/prismlauncher/prismlauncher/), and information about translating PrismLauncher is available at <https://github.com/prismlauncher/Translations>
```

File: src/wiki/overview/code-of-conduct.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement via email at
-[polymc-enforcement@scrumplex.net](mailto:polymc-enforcement@scrumplex.net) (Email address subject to change).
+[prismlauncher-enforcement@scrumplex.net](mailto:prismlauncher-enforcement@scrumplex.net) (Email address subject to change).
```
Comment (Lutitious) [inline src/wiki/overview/code-of-conduct.md:73]: This email has not yet been created but i have let scrumplex know about it

**Review by Lutitious — CHANGES_REQUESTED** [review]

File: src/wiki/overview/code-of-conduct.md (hunk 2 above)
Comment (Lutitious) [reply src/wiki/overview/code-of-conduct.md:73]: ok this is the email address being used for this now

**Comment by Lutitious** [comment]
@jackmawer issue 4 and 5 resolved by these changes pls add ty

**Comment by lukeb2e** [comment]
Hey guys, you know your website is already online in the old version on `https://prismlauncher.org/`? Maybe you should already merge this and continue working on a new PR. At least I hope that link directs to your version.

**Comment by xemulat** [comment]
this should be merged...

**Comment by timoreo22** [comment]
Merging this just to get this at least somewhat in order, will be fixed in a later pr

**Comment by txtsd** [comment]
Good! But:

```
Naming guidelines:
Display Name: Prism Launcher
Slug (package name): prismlauncher
Alternative Slug: PrismLauncher
App ID: org.prismlauncher.PrismLauncher 
```

**Pull request merged**